transformer_engine/pytorch/attention.py

```diff
@@ -3286,6 +3293,7 @@ def forward(
                     self.attention_dropout if self.training else 0.0,
                     softmax_scale=self.softmax_scale,
                     causal="causal" in attn_mask_type,
+                    softcap=self.softcap,
```

Comment:
Shouldn't it be added to the fa_optional_forward_kwargs instead? Otherwise the previous versions of FA will complain.

Reply:
you are right, it is my fault